rebasehelper/output_tool.py

```diff
@@ -64,6 +65,8 @@ def print_cli_summary(cls, app):
             # checkers provide output only after successfully built packages
             if AbiCheckerTool.abi_changes:
                 ConsoleHelper.cprint('\nABI changes occured. Check abipkgdiff output', color='red')
+            if LicenseCheckTool.license_changes:
```

Comment:
@nforro Do you think it is appropriate to inform about license changes even after a failed build? I suppose it is.

Reply:
Definitely, that's actually the main reason for running different checkers at different phases of a rebase.